docs/operational_guide/upgrading_m3.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading M3
+
+## Overview
+
+This guide explains how to upgrade (and downgrade??) M3 from one version to another (e.g. from 0.14.0 to 0.15.0) - both on Kubernetes as well as on bare metal.
+This includes upgrading:
+
+- m3dbnode
+- m3coordinator (coming soon!)
+- m3query (coming soon!)
+- m3aggregator (coming soon!)
```

Comment:
This seems like the kind of thing that could say "coming soon" forever. Let's leave these out and say the same general steps apply (get a different binary in place), and we may add more info in the future about specific components.